scripts/src/main/scala/com/gu/mediaservice/scripts/EsScript.scala

```diff
@@ -211,6 +207,38 @@ object UpdateMapping extends EsScript {
   }
 }
 
+object GetMapping extends EsScript {
+
+  def run(esHost: String, extraArgs: List[String]) {
+    // TODO: add the ability to update a section of the mapping
+    object EsClient extends ElasticSearchClient {
+      val imagesAlias = "writeAlias"
+      val port = esPort
+      val host = esHost
+      val cluster = esCluster
+
+      def getMappings(specifiedIndex: Option[String]) {
+        val index = getCurrentAlias.getOrElse(imagesAlias)
+        println(s"getting mapping on index: $index")
+        val result = client.admin.indices
+          .prepareGetMappings(index)
+          .execute.actionGet
+
+        println(result.getMappings.get(index).get(imageType).getSourceAsMap)
```

Comment:
Could the code inside this println throw an exception  if the ES request above fails?